docker/entrypoint.sh

```diff
@@ -34,11 +34,12 @@ initialize_system() {
   APP_URL=${APP_URL:-http://localhost}
   APP_KEY=${APP_KEY:-SECRET}
 
-  DB_DRIVER=${DB_DRIVER:-mysql}
-  DB_HOST=${DB_HOST:-mysql}
+  DB_DRIVER=${DB_DRIVER:-pgsql}
+  DB_HOST=${DB_HOST:-127.0.0.1}
```

Comment:
This should default to `postgres` rather than `127.0.0.1` as we have name the postgres container and setup a Docjer link to the Cachet container in https://github.com/CachetHQ/Docker/blob/master/docker-compose.yml#L16